Formula/makensis.rb

```diff
@@ -11,6 +11,10 @@ class Makensis < Formula
     sha256 "2b802ba4f889a4add1ca35d16c0e9a291afde7f55fb3cea6b47d5a637a4d3934" => :yosemite
   end
 
+  # Build makensis with advanced logging of all installer actions
+  # From http://nsis.sourceforge.net/Special_Builds#Advanced_logging
+  option "with-advanced-logging", "Include advanced logging features that allow creating an exact log of all installer actions"
```

Comment:
It's true that the NSIS developers traditionally had an eye on overhead. It might make little sense in these days, but I still kind of respect them for it.